content/source/docs/enterprise/private/migrate.html.md

```diff
@@ -9,6 +9,24 @@ sidebar_current: "docs-enterprise2-private-installer-migration"
 This document outlines the procedure for migrating from the AMI-based Private Terraform Enterprise (PTFE)
 to the Installer-based PTFE.
 
+## Backup
+
+Before beginning, it's best create an additional backup of your RDS database. This will allow you to rollback the data and continue to use the AMI if necessary.
+
+To do so, go to the [Amazon RDS Instances](https://console.aws.amazon.com/rds/home?region=us-east-1#dbinstances:). You may need to change the region that you are viewing in order to see your PTFE instance. Once you find it, click on the instance name. On the next page, select **Instance Actions** and then **Take snapshot**. On the **Take DB Snapshot** page, enter a name for the snapshot such as _pre installer migration_, and then click **Take Snapshot**.
```

Comment:
To do so > To create an RDS backup

Conventionally, I think we usually put literal text entry in `backticks`, so it would read:

"enter a name for the snapshot, such as `Pre-installer Migration`, and then click...